CHANGES.rst

```diff
@@ -6,10 +6,41 @@
 Summary
 +++++++
 
-- The Gammapy v0.11 release is planned for March 2019.
+- Released on March 29th, 2019
```

Comment:
Remove "th" to be consistent with the rest.

Reply:
Done